app/Providers/RouteServiceProvider.php

```diff
@@ -56,13 +60,15 @@ public function boot()
                 ->firstOrFail();
         });
 
-        Route::bind('activity', function ($value, $route) {
+        Route::bind('activity', function ($value, $route) use ($ID_hasher) {
```

Comment:
![MAJOR](https://sonarsource.github.io/sonar-github/severity-major.png 'Severity: MAJOR') Remove the unused function parameter "$route". [![rule](https://sonarsource.github.io/sonar-github/rule.png)](https://sonarcloud.io/coding_rules#rule_key=php%3AS1172)